tools/patch-swc-wasm.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+const fs = require('node:fs');
+const path = require('node:path');
+
+const base64 = fs.readFileSync(path.join(__dirname, '../deps/swc/wasm_bg.wasm')).toString('base64');
+const outputFile = path.join(__dirname, '../deps/swc/wasm.js');
+const originalContent = fs.readFileSync(outputFile);
+
+fs.writeFileSync(outputFile, `'use strict'\n
+const { Buffer } = require('node:buffer');\n
```

Comment:
@kdy1 I created this script to insert the wasm as byte array in the `wasm.js` file because we cannot perform a readFile or require

Reply:
I inlined the wasm binary. https://www.npmjs.com/package/@swc/wasm-typescript/v/1.6.8-nightly-20240704.3?activeTab=code